packages/opentelemetry-core/src/common/time.ts

```diff
@@ -19,7 +19,7 @@ import { otperformance as performance } from '../platform';
 import { TimeOriginLegacy } from './types';
 
 const NANOSECOND_DIGITS = 9;
-const SECOND_TO_NANOSECONDS = Math.pow(10, NANOSECOND_DIGITS);
+export const SECOND_TO_NANOSECONDS = Math.pow(10, NANOSECOND_DIGITS);
```

Comment:
As as a small aside, would it make sense to have these be literals like `1e9` instead of a call to `Math.pow(`? The TypeScript compiler doesn't turn it into a literal, so it could become more code in the end browser client (see http://www.typescriptlang.org/play/?ssl=1&ssc=1&pln=1&pc=17#code/LIQwLgFgdADg9gdwBQEYAMAaABATgJQDcQA)

Reply:
it is not exported / changed anymore now